tests/cross-platform/.gitignore

```diff
@@ -1,2 +1,4 @@
-/dev
-/work
\ No newline at end of file
+/dev*
```

Comment:
Why the trailing `*`?

Reply:
this is a best-practice for MSYS2/Windows, because deleting will take so much time, so we just rename it to dev_ before deleting it at background.
it is also usefull to backup the current dev workspace to get a clean one.

this is just a nice to have, not a strict requirement.

Reply:
this is a best-practice for MSYS2/Windows, because deleting will take so much time, so we just rename it to dev_ before deleting it at background.
it is also usefull to backup the current dev workspace to get a clean one.

this is just a nice to have, not a strict requirement.

Reply:
I see.